plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/model/GeneratedJavaFileAccess.xtend

```diff
@@ -28,24 +26,26 @@ class GeneratedJavaFileAccess extends JavaFileAccess {
 			throw new IllegalArgumentException("It's always generated");
 	}
 
+	/**
+	 * prepends the addition of required imports of the employed annotations
+	 */
 	override getContent() {
-		val classAnnotations = annotations + codeConfig.classAnnotations.filter[appliesTo(this)]
-		classAnnotations.forEach[importType(annotationImport)]
-		val sortedImports = Lists.newArrayList(imports.values)
-		Collections.sort(sortedImports)
-		return '''
-			�codeConfig.fileHeader�
-			package �javaType.packageName��IF appendSemicolons�;�ENDIF�
-			
-			�FOR importName : sortedImports�
-				import �importName��IF appendSemicolons�;�ENDIF�
-			�ENDFOR�
-			
-			�typeComment�
-			�FOR annot : classAnnotations�
-				�annot.generate()�
-			�ENDFOR�
-			�internalContents�
-		'''
-	}	
+		classAnnotations.forEach[
+			importType(annotationImport)
+		]
+		super.getContent()
+	}
+	
+	private def getClassAnnotations() {
+		annotations + codeConfig.classAnnotations.filter[appliesTo(this)]
+	}
+	
+	override getInternalContent() '''
```

Comment:
In fact I'd prefer a name that describes the result of the method, e.g. getTopLevelClass